src/pages/payment.js

```diff
@@ -43,7 +45,12 @@ const PaymentPage = () => {
             marginTop: '7rem',
           }}
         >
-          {paymentStatus ? <SuccessCard /> : <PaymentCard />}
+          <PaymentCardContainer>
+            <Heading4 font='Inter' style={{ textAlign: 'center', maxWidth: '500px' }}>
+              Registration for non-NITR students has been closed for this year. Please come back
```

Comment:
Change the text to "Registration for NON-NITR Students has been closed. Please reach out to INNOVISION team in case of any query"